src/classes/audioContextWrapper.tsx

```diff
@@ -121,6 +127,20 @@ class AudioContextWrapper {
     this.waveform = newWaveform;
   }
 
+  setWavetable(newWavetable: Wavetable) {
+    this.wavetable = newWavetable;
+  }
+
+  setWavetableName(newWavetableName: string) {
+    var wavetableFromCache = WavetableCache.getSingleton().get(newWavetableName);
```

Comment:
```suggestion
    const setWavetableNotFound = () => {
          console.log("Unable to find wavetable " + newWavetableName + " in the cache");
          return Wavetable.createEmpty();
    }
    const waveTableFromCache = WavetableCache.getSingleton().get(newWavetableName) ?? setWaveTableNotFound()
```

Reply:
Above is a more Javascripty way of writing this, where you're never redeclaring anything. `const` lyf, baybeeeee